### PR TITLE
Map reading

### PR DESCRIPTION
**Pull request by seanybaggins**

### CODE REVIEW

File: guidance/rust_code/src/graph/mod.rs [hunk 1]
```diff
@@ -0,0 +1,346 @@
+use std::fs::File;
+use std::io::{prelude::*, BufReader, Seek, SeekFrom};
+
+#[derive(PartialEq, PartialOrd, Debug)]
```
Comment (dstockhouse) [inline guidance/rust_code/src/graph/mod.rs]: Is it valid for GPSPoint to derive PartialOrd? In what situation would you compare which point is "greater" than another point?

File: guidance/rust_code/src/graph/mod.rs [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+#[derive(Debug)]
+pub struct Vertex {
+    // Will be used to display key locations to UI
+    name: String,
+
+    // Will be used to identify adjacent nodes and edges
+    pub(self) gps_point: GPSPoint,
+
+    // Key data to determine the shortest path using Dijkstra's Algorithm
```
Comment (dstockhouse) [inline guidance/rust_code/src/graph/mod.rs:26]: Are these necessary if the Graph struct performs Dijkstra's algorithm?

File: guidance/rust_code/src/graph/mod.rs (hunk 1 above)
Comment (seanybaggins) [reply guidance/rust_code/src/graph/mod.rs]: Removed partial ord

File: guidance/rust_code/src/graph/mod.rs (hunk 2 above)
Comment (seanybaggins) [reply guidance/rust_code/src/graph/mod.rs:26]: This is okay. The self keyword means gps_point is only public to modules within the same directory. In this case, the unit tests.

**Pull request merged**